docs/sources/administration/data-source-management/index.md

```diff
@@ -111,21 +114,13 @@ You can make a panel retrieve fresh data more frequently by increasing the **Max
 
 ### Data sources that work with query caching
 
-Query caching works for all [Enterprise data sources](/grafana/plugins/?type=datasource&enterprise=1) as well as the following [built-in data sources]({{< relref "../../datasources/" >}}):
-
-- CloudWatch Metrics
-- Google Cloud Monitoring
-- InfluxDB
-- Microsoft SQL Server
-- MySQL
-- Postgres
-- Tempo
-
-Some data sources, such as Elasticsearch, Prometheus, and Loki, cache queries themselves, so Grafana query caching does not improve performance.
+Query caching works for all backend data sources. More specifically, caching works with Grafana's [built-in data sources]({{< relref "../../datasources/#built-in-core-data-sources" >}}), and any [backend data source plugins](https://grafana.com/grafana/plugins/?type=datasource) that extend the `DataSourceWithBackend` class in the plugins SDK.
 
-Query caching also works for all data sources that include a backend. More specifically, caching works with data sources that extend the `DataSourceWithBackend` class in the plugins SDK.
+To verify that a data source works with query caching, follow the instructions below to **Enable and Configure query caching**. If caching is enabled in Grafana but the Caching tab is not visible for the given data source, then query caching is not available for that data source.
 
-To tell if a data source works with query caching, follow the instructions below to **Enable and Configure query caching**. If caching is enabled in Grafana but the Caching tab is not visible for the given data source, then query caching is not available for that data source.
+{{% admonition type="note" %}}
+Some data sources, such as Elasticsearch, Prometheus, and Loki, cache queries themselves, so Grafana query caching does not significantly improve performance.
```

Comment:
I'm not sure if this is too niche to include in the docs, but Loki (and perhaps the others) do not cache resource requests, so adding resource caching can be really helpful if they have large variable queries on panels.

Reply:
Maybe we could find a way to say this in the #caching-benefits section? We could say something about how resource caching avoids unnecessary duplicate requests for static data, and maybe give an example about Loki. Feel free to take a stab at it

Reply:
Thinking more about this - we now link to the backend plugins page which covers the use cases for resource caching, and the benefits of resource caching are covered in the benefits section. I think we are probably okay leaving out the loki detail. WDYT?

Reply:
Ah good point! Maybe we could point it out here like this?
```
Some data sources, such as Elasticsearch, Prometheus, and Loki, cache queries themselves, so Grafana **query** caching does not significantly improve performance, though **resource** caching may help. See [these docs](<insert backend plugins page>) for details.
```
There has been some confusion in the past on this with some larger customers, so I could see it being useful, even if it's a bit redundant. Wdyt?

Reply:
I like it, how is this: [e61bc48](https://github.com/grafana/grafana/pull/70556/commits/e61bc48466d1c372e644d6cf04182a30bc495da3)